pkg/operator/operator.go

```diff
@@ -142,11 +144,13 @@ func New(
 	optr.proxyLister = proxyInformer.Lister()
 	optr.proxyListerSynced = proxyInformer.Informer().HasSynced
 
-	optr.validatingWebhookCache = resourceapply.NewResourceCache()
+	optr.validatingMachineWebhookCache = resourceapply.NewResourceCache()
+	optr.validatingMetal3RemediationWebhookCache = resourceapply.NewResourceCache()
```

Comment:
I don't remember much about these caches, they can't be shared?

Reply:
Good question... I thought so, because both old configs had their own cache already.
But looking at the cache code I think all resources can use the same cache... 🤔